artemis-jdbc-store/src/main/java/org/apache/activemq/artemis/jdbc/store/file/JDBCSequentialFile.java

```diff
@@ -75,33 +57,30 @@
 
    private JDBCSequentialFileFactory fileFactory;
 
-   private int maxSize;
+   private final Object writeLock;
 
-   private SQLProvider sqlProvider;
+   private final JDBCFileFactoryDriver dbDriver;
 
-   private final Object writeLock;
+   private static final Logger log = Logger.getLogger(JDBCSequentialFile.class.getName());
```

Comment:
It seems that this was changed later on another commit...

It gets really confusing to merge committs like this though. I would prefer these squashed but that's just my preference